liferay-echo/liferay-portal
# LPS-121885 Migrate v2 usages in layout/layout-portlets-web

The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

![](https://user-images.githubusercontent.com/5572/109981784-884b2f00-7d01-11eb-9c6f-4c4b5e3e7901.gif)

**Test plan**:
- Navigate to **Site Builder** > **Widgets**
- Make sure you can filter and order Widgets
